source/3-our-code-process.md

```diff
@@ -11,10 +11,10 @@ We use ember.js for the front-end of our apps and Rails for the back-end code an
 We follow a standard branching workflow during development:
 
 1. A developer **creates a branch** from master that describes the feature. The branch is prefixed with the developer initials. Example: 'cb-allow-users-to-order-takeout'
-1. During development, the branch should be periodically pushed to origin. When the code is complete with **tests written to back up the change**, the developer looks through the commit messages in the branch and makes sure that they are readable and understandable. Any commit messages that need to be updated are fixed using the interactive rebase feature of git. 
+1. During development, the branch should be periodically pushed to origin. When the code is complete with **tests written to back up the change**, the developer looks through the commit messages in the branch and makes sure that they are readable and understandable. Any commit messages that need to be updated are fixed using the interactive rebase feature of git.
 1. The developer **creates a Pull Request** on GitHub. Slack notifies the team. We require all Pull Requests to be reviewed and marked with a +1 by another team member. This marking indicates that the changes are ready to merge to master.
 1. When the Pull Request is opened, **a CI (Continuous Integration) server runs the test suite**, ensuring the entire test suite still passes. The CI server will automatically update the Pull Request with the pass/fail status of the test suite.
-1. If a Pull Request has a +1 and passes CI, it is **merged to master** by the developer who originally opened it. 
+1. If a Pull Request has a +1 and passes CI, it is **merged to master** by the developer who originally opened it.
```

Comment:
Instead of using +1, which is just developer speak - maybe something like:

1. If a pull request has been reviewed and approved, passes CI, it is merged to master by the developer who originally opened it.